layers/core_checks/cc_descriptor.cpp

```diff
@@ -826,22 +826,22 @@ bool CoreChecks::ValidateDrawState(const DescriptorSet &descriptor_set, uint32_t
                                             const_cast<DescriptorSet &>(descriptor_set), set_index, framebuffer, loc);
 
     for (const auto &binding_pair : bindings) {
```

Comment:
```suggestion
    for (const auto &[desc_set_binding_i, desc_set_reqs] : bindings) {
```
and bindings should be renamed, (actually its type, `BindingVariableMap`, should too) to something like `desc_set_binding_to_requirements_map`

Reply:
overall, the issue is everything slowly was becoming `binding` because it works for so many things in this area of code

Reply:
also this syntax is much better than `.first`/`.second`!